From 0c6d0c39ec10845d0869420ac43cf0409e035106 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 13:49:41 +0800
Subject: [PATCH 1397/1594] MLK-10783-7 doc: dt-binding: mxs-phy: add
 compatible string "fsl,imx6ul-usbphy"

commit 45583aa93ee7d3f8e930854603121f7facdcffb9 from
git://git.freescale.com/imx/linux-2.6-imx.git

add compatible string "fsl,imx6ul-usbphy"

Signed-off-by: Peter Chen <peter.chen@freescale.com>
---
 Documentation/devicetree/bindings/usb/mxs-phy.txt |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Documentation/devicetree/bindings/usb/mxs-phy.txt b/Documentation/devicetree/bindings/usb/mxs-phy.txt
index a3b6055..8f8cbb3 100644
--- a/Documentation/devicetree/bindings/usb/mxs-phy.txt
+++ b/Documentation/devicetree/bindings/usb/mxs-phy.txt
@@ -7,6 +7,7 @@ Required properties:
 	* "fsl,imx6sl-usbphy" for imx6sl
 	* "fsl,vf610-usbphy" for Vybrid vf610
 	* "fsl,imx6sx-usbphy" for imx6sx
+	* "fsl,imx6ul-usbphy" for imx6ul
   "fsl,imx23-usbphy" is still a fallback for other strings
 - reg: Should contain registers location and length
 - interrupts: Should contain phy interrupt
-- 
1.7.5.4

